gulpfile.babel.js

```diff
@@ -75,6 +75,17 @@ function onBuild( done ) {
 				} );
 		}
 
+		// uglify some other random files
+		gulp.src( [ '_inc/*.js', '!_inc/*.min.js' ] )
+			.pipe( sourcemaps.init() )
```

Comment:
Can we make sure we only generate sourcemaps when it's a development, and not a production build?

Reply:
So should we also eliminate all the CSS .map files too? We already output those for production builds.